Bottom: 564d5e3cf91a6124ee1713024bd2cd72780f2749
Top:    3ac1ed3f4541c6df5b1e14b36ae286294be51842
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-17 19:59:09 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
